libs/live-dmk/src/index.tsx

```diff
@@ -1,19 +1,34 @@
-import React, { createContext, useContext, useEffect, useState } from "react";
+import React, { createContext, useCallback, useContext, useEffect, useState } from "react";
 import Transport from "@ledgerhq/hw-transport";
 import {
   DeviceManagementKitBuilder,
   ConsoleLogger,
-  type DeviceManagementKit,
+  DeviceManagementKit,
   type DeviceSessionState,
   DeviceStatus,
+  DeviceBusyError,
   LogLevel,
-  BuiltinTransports,
+  DiscoveredDevice,
+  DeviceModelId,
```

Comment:
[COULD] maybe alias to `DMKDeviceModelId` and the other one to `LLDeviceModelId` for better clarity